daemon/container.go

```diff
@@ -822,10 +843,15 @@ func (container *Container) monitorExec(execConfig *execConfig, callback execdri
 }
 
 func (c *Container) Attach(stdin io.ReadCloser, stdout io.Writer, stderr io.Writer) chan error {
-	return attach(&c.StreamConfig, c.Config.OpenStdin, c.Config.StdinOnce, c.Config.Tty, stdin, stdout, stderr)
+	err := attach(&c.StreamConfig, c.Config.OpenStdin, c.Config.StdinOnce, c.Config.Tty, stdin, stdout, stderr)
```

Comment:
I didn't understand this change. Btw it returns `chan`, so now it's super-confusing.